Library/Homebrew/shims/super/ant

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
+
 export HOMEBREW_CCCFG="O$HOMEBREW_CCCFG"
+
 ant=/usr/bin/ant
-[ -x "$ant" ] || ant="$(${HOMEBREW_BREW_FILE} --prefix ant)/bin/ant"
+[[ -x "$ant" ]] || ant="$("$HOMEBREW_BREW_FILE" --prefix ant)/bin/ant"
```

Comment:
True. Note that I avoided all changes to functionality, however subtle, and focused on the style issues only, so not addressing this was intentional.